saleor/core/utils/random_data.py

```diff
@@ -977,7 +977,7 @@ def _create_staff_user(staff_password, email=None, superuser=False):
     if not email:
         email = get_email(first_name, last_name)
 
-    staff_user = User.objects.filter(email=email).first()
+    staff_user = User.objects.filter(email=email, is_staff=True).first()
```

Comment:
It means that you will try to create staff user, for the email that is already present in the DB. Means that this will raise unhandled exception. 
For example:
- In DB, you have `User(email="test@example.com", is_staff=False)`
- `User.objects.filter(email="test@example.com", is_staff=True).first()` returns None
- Then call `User.objects.create_user(` which will raise an exception, as user with the given email is already present DB.

IMHO, the logic should be modified to handle the case, when user is present, but is marked as `is_staff=False`